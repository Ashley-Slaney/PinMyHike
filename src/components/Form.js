export const Form = () => {
  return (
    <div className="form">
      <textarea id="description" type="text" cols="40" rows="5"
        placeholder="Enter a description of your hike" className="text-field">
      </textarea>
      <button type="submit" className="btn">Submit</button>
    </div>
  )
}

export default Form