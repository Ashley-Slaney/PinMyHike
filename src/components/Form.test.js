import renderer from 'react-test-renderer'
import Form from './Form'

it('renders correctly when there are no props', () => {
  const tree = renderer.create(<Form />).toJSON()
  expect(tree).toMatchSnapshot()
})
