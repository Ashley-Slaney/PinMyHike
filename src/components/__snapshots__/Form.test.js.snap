// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no props 1`] = `
<div
  className="form"
>
  <textarea
    className="text-field"
    cols="40"
    id="description"
    placeholder="Enter a description of your hike"
    rows="5"
    type="text"
  />
  <button
    className="btn"
    type="submit"
  >
    Submit
  </button>
</div>
`;
